content/en/docs/components/serving/kfserving.md

```diff
@@ -52,23 +72,36 @@ We frequently add examples to our [GitHub repo](https://github.com/kubeflow/kfse
 * Join our [working group](https://groups.google.com/forum/#!forum/kfserving) for meeting invitations and discussion.
 * [Read the docs](https://github.com/kubeflow/kfserving/tree/master/docs).
 * [API docs](https://github.com/kubeflow/kfserving/tree/master/docs/apis/README.md).
+* [Debugging guide](https://github.com/kubeflow/kfserving/blob/master/docs/KFSERVING_DEBUG_GUIDE.md)
 * [Roadmap](https://github.com/kubeflow/kfserving/tree/master/ROADMAP.md).
 * [KFServing 101 slides](https://drive.google.com/file/d/16oqz6dhY5BR0u74pi9mDThU97Np__AFb/view).
 
-## Prerequisites
-Knative Serving (v0.8.0 +) and Istio (v1.1.7+) should be available on your Kubernetes cluster.
+## Standalone KFServing
+### Install Knative/Istio
+Knative Serving (v0.11.2 +) and Istio (v1.1.7+), Cert Manager(v0.12.0+) should be available on your Kubernetes cluster.
 
-Read more about [installing Knative on a Kubernetes cluster](https://github.com/kubeflow/kfserving/blob/master/docs/DEVELOPER_GUIDE.md#install-knative-on-a-kubernetes-cluster).
+* [installing Knative on a Kubernetes cluster](https://knative.dev/docs/install/).
+* [installing Istio for Knative](https://knative.dev/docs/install/installing-istio/).
+* [installing Cert Manager](https://cert-manager.io/docs/installation/kubernetes/)
+* [Configure secure HTTPS](https://knative.dev/docs/serving/using-a-tls-cert/)
+* [Setup custom domain](https://knative.dev/docs/serving/using-a-custom-domain/)
```

Comment:
My suggestion is the above are too much details for KFServing official docs? If needed, they make more sense in the github repo - and ideally only be pointed in context? Ideally we should point to one link for KNative section in our github if multiple links need to be traced?